packages/rest-hooks/src/manager/DevtoolsManager.ts

```diff
@@ -24,6 +24,25 @@ export default class DevToolsManager implements Manager {
   constructor(
     config: DevToolsConfig = {
       name: `Rest Hooks: ${globalThis.document?.title}`,
+      serialize: {
+        replacer: (key: string, value: unknown) => {
```

Comment:
seems kind of weird that the return type of this function is `string | Date | unknown`. Should the date and/or the value be stringified?

Reply:
in the case it's not what we're processing we just want to pass through. This has to operate on any value that comes in. I could make it a generic but since I don't really care about return value, I don't see why it's helpful.

Reply:
I think devtools handles other serializations itself https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md

Reply:
fair enough, so it's kind of like toJSON

Reply:
Exactly. In fact I believe at some point it uses this API https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#the_replacer_parameter